src/main/webapp/index.jsp

```diff
@@ -20,7 +20,7 @@
       <!--Site Header-->
       <header>
           <nav>
-              <div id="userpage-icon">
+              <div id="user-icon">
```

Comment:
nit: Maybe it's better to call the `div` element as `dashboard-icon-container`, since it contains icons instead of being the icon itself.

Reply:
Agreed! That sounds better